client/js/options.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const $ = require("jquery");
+const escapeRegExp = require("lodash/escapeRegExp");
```

Comment:
It adds nothing to the size because client/js/libs/handlebars/ircmessageparser/findChannels.js already has a reference to it.